proto/axelar/evm/v1beta1/service.proto

```diff
@@ -153,18 +153,25 @@ service QueryService {
     option (google.api.http).get = "/axelar/evm/v1beta1/chains";
   }
 
+  // Command queries the command of a chain provided the command id
+  rpc Command(CommandRequest) returns (CommandResponse) {
+    option (google.api.http).get = "/axelar/evm/v1beta1/command_request";
```

Comment:
```suggestion
    option (google.api.http).get = "/axelar/evm/v1beta1/command/{chain}/{id}";
```

Reply:
did that originally, but `make proto-all` generates syntactically incorrect code in `/Users/steliosdaveas/Projects/axelar/axelar-core/x/evm/types/service.pb.gw.go` trying to assign to field with name `Id` instead of `ID`:

```golang
val, ok = pathParams["id"]
if !ok {
	return nil, metadata, status.Errorf(codes.InvalidArgument, "missing parameter %s", "id")
}

protoReq.Id, err = runtime.String(val) // <-- error here, field name is protoReq.ID
```

seems that the service auto generated code doesn't take into consideration aliases in the [protobuf definition](https://github.com/axelarnetwork/axelar-core/blob/01485f28e5084b6470be5e867990d6cb1d87ec77/proto/axelar/evm/v1beta1/query.proto#L134).

```proto
message CommandRequest {
  string chain = 1;
  string id = 2 [ (gogoproto.customname) = "ID" ];
}
```

the generated code is correct if remove `[ (gogoproto.customname) = "ID" ]` from the protobuf definition.

Reply:
ah, you can ignore it for now then